### PR TITLE
Added links for the getter function.

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Review by jamesray1 — COMMENTED** [review]
Changed the URL for getter to a ref.

File: docs/types.rst [hunk 1]
```diff
@@ -557,7 +557,7 @@ So ``bytes`` should always be preferred over ``byte[]`` because it is cheaper.
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters!
 
-It is possible to mark arrays ``public`` and have Solidity create a getter.
+It is possible to mark arrays ``public`` and have Solidity create a :ref:`getters <visibility-and-getters>`.
```
Comment (chriseth) [inline docs/types.rst]: Can you change that to `getter` (singular), please?

File: docs/types.rst [hunk 2]
```diff
@@ -806,7 +806,7 @@ Because of this, mappings do not have a length or a concept of a key or value be
 Mappings are only allowed for state variables (or as storage reference types
 in internal functions).
 
-It is possible to mark mappings ``public`` and have Solidity create a getter.
+It is possible to mark mappings ``public`` and have Solidity create a :ref:`getters <visibility-and-getters>`.
```
Comment (chriseth) [inline docs/types.rst]: Can you change that to `getter` (singular), please?

**Review by chriseth — CHANGES_REQUESTED** [review]

**Comment by jamesray1** [comment]
@chriseth, I updated to getter (singular).

**Comment by jamesray1** [comment]
@chriseth

**Comment by chriseth** [comment]
Thanks, this works!

**Pull request merged**